drink-cli/src/app_state/mod.rs

```diff
@@ -49,8 +53,8 @@ pub struct UiState {
 
 impl UiState {
     pub fn new(pwd_override: Option<PathBuf>) -> Self {
-        let pwd = pwd_override.unwrap_or_else(
-            || env::current_dir().expect("Failed to get current directory"));
+        let pwd = pwd_override
```

Comment:
nit :) I think you may want to rename it to wd or cwd as pwd is `print working directory`